portal-frontend/src/app/features/create-application/create-application.component.html

```diff
@@ -0,0 +1,47 @@
+<mat-stepper linear='true' #stepper>
+  <mat-step label='Application' [stepControl]='firstForm'>
+    <div class='header'>
+      <h6>Application ID: 66112 | Trans, Util & Rec</h6>
```

Comment:
Is this a placeholder for the next ticket?

Reply:
Yes, I figured we could un-hardcode this as we add the types and getting file number from ALCS